app/views/member_profiles/passwords/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_for(@user, :url => { :action => "update" } ) do |f| %>
+  <div class="field">
+    <%= f.label :password, "Password" %><br />
+    <%= f.password_field :password, :autocomplete => "off"  %>
+  </div>
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation %>
+  </div>
+  <div class="action_container">
```

Comment:
I don't see the class `action_container` mentioned in the CSS anywhere. I might have missed it, though. What does it do?